packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -1145,8 +1145,7 @@ class _LineItemState extends State<LineItem>
     // ignore: avoid-unused-parameters
     required bool Function() isHoverStale,
   }) async {
-    final isPaused =
-        serviceManager.isolateManager.mainIsolateState?.isPaused.value ?? false;
+    final isPaused = serviceManager.isMainIsolatePaused;
     if (!isPaused) return null;
```

Comment:
nit: since this is simple now, just write `if (serviceManager.isMainIsolatePaused)` directly